app/templates/components/component-tree-item.hbs

```diff
@@ -21,10 +20,75 @@
   {{/if}}
 
   <code
-    class="component-tree-item__tag {{if @item.isComponent "component-tree-item__bracket"}} {{if (not @item.hasChildren) "component-tree-item__bracket--self-closing"}}"
+    class="component-tree-item__tag
+      {{if
+        @item.isComponent
+        (if
+          @item.isCurlyInvocation
+          "component-tree-item-classic__bracket"
+          (if
+            @item.hasChildren
+            "component-tree-item__bracket--self-closing"
+            "component-tree-item__bracket"
```

Comment:
Oops. My bad, these last two are in the reverse order I think (if hasChildren -> *not* self-closing)